src/test/java/com/icthh/xm/gate/web/rest/errors/ExceptionTranslatorIntTest.java

```diff
@@ -1,109 +0,0 @@
-package com.icthh.xm.gate.web.rest.errors;
-
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
-
-import com.icthh.xm.commons.exceptions.ErrorConstants;
-import com.icthh.xm.commons.exceptions.spring.web.ExceptionTranslator;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.http.MediaType;
-import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.test.web.servlet.MockMvc;
-import org.springframework.test.web.servlet.setup.MockMvcBuilders;
-
-import com.icthh.xm.gate.GateApp;
-import com.icthh.xm.gate.config.SecurityBeanOverrideConfiguration;
-
-/**
- * Test class for the ExceptionTranslator controller advice.
- *
- * @see ExceptionTranslator
- */
-@RunWith(SpringRunner.class)
-@SpringBootTest(classes = {GateApp.class, SecurityBeanOverrideConfiguration.class})
-public class ExceptionTranslatorIntTest {
```

Comment:
Test should not be deleted, as third clients are using JSON response with defined structure for Exceptions